ci/concourse/run-tests.sh

```diff
@@ -1,17 +1,14 @@
 #!/bin/bash
 
-# Fail on errors
-set -euxo pipefail
+set -euo pipefail
 
-. "$(dirname $0)"/env.sh
+/ppsspp/build-sdl/PPSSPPHeadless rust-build-dir/EBOOT.PBP --timeout=10 -r .
```

Comment:
Switching back to this does mean that local runs of CI will no longer work, is that intended? I use `./ci/concourse/build-rust.sh && ./ci/concourse/run-tests.sh` as a pre-flight check before committing core changes. Can switch back to just CI if it's fast now but I tend to prefer local runs being an option.

Reply:
(answered on Discord)